### PR TITLE
Fix to keyfax session, use create method

### PR DESCRIPTION
**Pull request by cdccollins**

### CODE REVIEW

File: spec/features/repair_request_spec.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ def stub_post_repair_request
     )
   end
 
-  def stub_keyfax_get_startup_url
+  def stub_keyfax_session_create
```
Comment (pudiva) [inline spec/features/repair_request_spec.rb]: Not that it is super important, but the old naming was better. This is a stub to the network call, not the `KeyfaxSession.create` method.

File: app/controllers/repair_requests_controller.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@ def create
         format.html { redirect_to work_order_path(@repair_request.work_orders.first.reference), notice: ['Repair raised!'] }
       else
         @cautionary_contacts = Hackney::CautionaryContact.find_by_property_reference(@property.reference)
-        @keyfax_session = Hackney::KeyfaxSession.get_startup_url(current_page_url: new_property_repair_request_url(@property.reference))
+        @keyfax_session = Hackney::KeyfaxSession.create(current_page_url: new_property_repair_request_url(@property.reference))
```
Comment (pudiva) [inline app/controllers/repair_requests_controller.rb:37]: I'm very worried this didn't make the tests crash. Can you please investigate and fix them?

File: spec/features/repair_request_spec.rb (hunk 1 above)
Comment (cdccollins) [reply spec/features/repair_request_spec.rb]: changed them back 👍

File: app/controllers/repair_requests_controller.rb (hunk 2 above)
Comment (cdccollins) [reply app/controllers/repair_requests_controller.rb:37]: we don't test the controllers, so should I add this to the feature spec?

Comment (pudiva) [reply app/controllers/repair_requests_controller.rb:37]: the feature spec should be crashing here

Comment (pudiva) [reply app/controllers/repair_requests_controller.rb:37]: I mean the feature spec is a browser that do the same thing as a user, so If it crashes for a user, then the test should as well.

**Review by pudiva — APPROVED** [review]
:shipit:

**Pull request merged**